### PR TITLE
Validates language uniqueness to avoid duplicates

### LINKED ISSUE #4096: Disable entering duplicate language options for casa orgs
### Impacted User Types
 - admins

### Environment
all

### Current Behavior
Multiple of the same language can be entered as language options for casa orgs
image

### Expected Behavior
A validation error is shown when duplicate languages are entered

### How to Replicate
ex: 
1. - Log as an admin.
2. - Click on "Edit Organization" in the left sidebar menu.
3. - Scroll to the languages section
4. - Copy a language from the list
5. - Click new language
6. - Paste the copied language into the box and submit

### Login Details:
[Link to QA site](https://casa-qa.herokuapp.com/)  

Login Emails: 
- casa_admin1@example.com view site as an admin

password for all users: 12345678  

### Questions? Join Slack!

We highly recommend that you join us in slack https://rubyforgood.herokuapp.com/ #casa channel to ask questions quickly and hear about office hours (currently Tuesday 6-8pm Pacific), stakeholder news, and upcoming new issues.

### PR DESCRIPTION
**Pull request by bugsmithd**
### What github issue is this PR for, if any?
Resolves #4096

### What changed, and why?
Language model and language model spec. Any changes in `lib` are done by `bundle exec standardrb --fix`.

### How will this affect user permissions?
- Volunteer permissions: No effect
- Supervisor permissions: No effect
- Admin permissions: No effect

### How is this tested? (please write tests!) 💖💪
Rspec

### Screenshots please :)

Screenshot 2023-01-20 at 1 07 52 AM

Screenshot 2023-01-20 at 1 10 34 AM

### Feelings gif (optional)
What gif best describes your feeling working on this issue? https://giphy.com/
How to embed:
`alt text`

### Feedback please? (optional)
We are very interested in your feedback! Please give us some :) https://forms.gle/1D5ACNgTs2u9gSdh9

### CODE REVIEW

**Comment by compwron** [comment]
Looks good except the failing tests :)

**Comment by anshu1992** [comment]
Yep, fixing them.

**Comment by anshu1992** [comment]
Hey @compwron, these tests are passing on my local machine. Do you know if these tests fail randomly?

File: app/javascript/application.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 /* global window */
 
+import './jQueryGlobalizer.js'
```
Comment (compwron) [inline app/javascript/application.js:3]: why the order change here?

File: app/controllers/case_contacts_controller.rb [hunk 2]
```diff
@@ -83,7 +83,7 @@ def create
     elsif @selected_cases.count > 1
       redirect_to case_contacts_path(success: true)
     else
-      redirect_to casa_case_path(CaseContact.last.casa_case, success: true)
+      redirect_to casa_case_path(CaseContact.last.casa_case, success: true), notice: "Case contact created successfully."
```
Comment (compwron) [inline app/controllers/case_contacts_controller.rb]: Is this correct? What is the rationale here?

File: spec/system/casa_org/edit_spec.rb [hunk 3]
```diff
@@ -16,6 +16,7 @@
     stub_twillio
     sign_in admin
     visit edit_casa_org_path(organization)
+    Faker::Nation.unique.clear # Clears used values for Faker::Nation
```
Comment (compwron) [inline spec/system/casa_org/edit_spec.rb:19]: Why is this needed?

File: spec/system/case_contacts/new_spec.rb [hunk 4]
```diff
@@ -128,8 +128,8 @@
       expect(page).to have_text("Read more")
       expect(page).to have_text(expected_text)
 
-      sleep(2)
-      click_on "Close" # close thank-you modal
+      # sleep(2)
```
Comment (compwron) [inline spec/system/case_contacts/new_spec.rb:131]: This is probably a problem?

File: app/javascript/application.js (hunk 1 above)
Comment (anshu1992) [reply app/javascript/application.js:3]: I changed the order because Bootstrap depends on jquery. Loading Jquery first, so other bootstrap features work correctly. This also fixes some console errors.

File: app/controllers/case_contacts_controller.rb (hunk 2 above)
Comment (anshu1992) [reply app/controllers/case_contacts_controller.rb]: The spec was checking for a success message to be received. There was none in actual action so added this to make the test pass.

File: spec/system/casa_org/edit_spec.rb (hunk 3 above)
Comment (anshu1992) [reply spec/system/casa_org/edit_spec.rb:19]: As soon as I added the uniqueness validation, this test started to fail. So I had to make sure that the names generated for the five language records are different. `Faker.unique` was throwing an error. I checked the faker documentation and found this [Ensuring unique values](https://randallreilly.atlassian.net/browse/STRG-903?atlOrigin=eyJpIjoiZWQzN2I0ZDA0NWMzNDAzMDk4NDU2Y2MwMmJjYmQ0ZTkiLCJwIjoiaiJ9).

File: spec/system/case_contacts/new_spec.rb (hunk 4 above)
Comment (anshu1992) [reply spec/system/case_contacts/new_spec.rb:131]: Sleep was not required here. I was unsure why this was put in the first place, so I commented it instead of completely removing it.

**Comment by anshu1992** [comment]
@compwron Fixes tests. The lint error which is failing is irrelevant of this code. I can't fix the linter check because the file is not even present in my PR.

**Review by compwron — APPROVED** [review]
Very nice :)
